Fix screen detection on maximize (add fallback)

After installing dfc_windows_0_8_0-beta.exe I get error screen right away, a previous 0.7 version did work (not sure but probably v0.7.7.2-alpha)
After reading some other issues I tried : 

1. deleting C:\ProgramData\FAForever\cache
2. starting as admin
3. deleting C:\ProgramData\FAForever
4. reinstalling several times
5. running at one screen at 1080p@100% scaling after cleaning window Size/Location cache with ccleaner

But problem remained. Running on Windows 10 Pro, version 1709
My laptop with Windows 10 Home, also 1709, runs this client fine.

It might have to do with my screen settings, I have 1 4k@150% screen and 2 1080p@100% screens.
I noticed before the java client gets confused when having screens at different scaling, but step 5 didn't fix this either.

error.log:
`Exception in Application start method
java.lang.RuntimeException: Exception in Application start method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$154(LauncherImpl.java:182)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at com.sun.javafx.collections.ObservableListWrapper.get(ObservableListWrapper.java:89)
	at com.faforever.client.fx.WindowController.getVisualBounds(WindowController.java:68)
	at com.faforever.client.fx.WindowController.maximize(WindowController.java:115)
	at com.faforever.client.fx.WindowController.maximize(WindowController.java:72)
	at com.faforever.client.main.MainController.display(MainController.java:337)
	at com.faforever.client.FafClientApplication.showMainWindow(FafClientApplication.java:97)
	at com.faforever.client.FafClientApplication.start(FafClientApplication.java:83)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$161(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$174(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$172(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$173(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$147(WinApplication.java:177)
	... 1 more`

--- a/src/main/java/com/faforever/client/fx/WindowController.java
+++ b/src/main/java/com/faforever/client/fx/WindowController.java
@@ -65,7 +65,8 @@ public class WindowController implements Controller<Node> {
 
     Rectangle2D windowCenter = new Rectangle2D(x1, y1, 1, 1);
     ObservableList<Screen> screensForRectangle = Screen.getScreensForRectangle(windowCenter);
-    return screensForRectangle.get(0).getVisualBounds();
+    Screen targetScreen = screensForRectangle.stream().findFirst().orElse(Screen.getPrimary());
+    return targetScreen.getVisualBounds();
   }
 
   public static void maximize(Stage stage) {

